releasenotes/notes/auth-policy-for-components-d576a28135a224db.yaml

```diff
@@ -0,0 +1,6 @@
+---
+features:
+  - |
+    Expose an `AuthPolicy` type to provide consistent API for any component that requires authentication.
```

Comment:
Need to update this to reflect the class name change.

Reply:
Fixed.